modules/nested-tabs-html/assets/js/frontend/handlers/nested-tabs-html.js

```diff
@@ -294,17 +293,21 @@ export default class NestedTabsHtml extends Base {
 	 */
 	reInitSwipers( event, content ) {
 		const swiperElements = content.querySelectorAll( `.${ elementorFrontend.config.swiperClass }` );
+
 		for ( const element of swiperElements ) {
 			if ( ! element.swiper ) {
 				return;
 			}
+
 			element.swiper.initialized = false;
 			element.swiper.init();
 		}
 	}
 
 	onInit( ...args ) {
-		this.createMobileTabs( args );
+		if ( elementorFrontend.isEditMode() ) {
+			this.setContainerAttributes( args );
```

Comment:
@hein-obox I think it would be better to set the container attributes in the `View`

Reply:
Let me try that!

Reply:
@louiswol94 Implemented.